GetFed/GetFed/Helpers/Identity.swift

```diff
@@ -31,5 +35,23 @@ enum Identity: String {
         }
     }
     
+    var cellID: String {
+        switch self {
+        case .foodSearchResultCell:
+            return "foodSearchResultCell"
+        default:
+            return ""
+        }
+    }
+    
+    var nibID: String {
```

Comment:
Perhaps you should separate out these constants so they all have the same properties. Or, just make the enum `String` based so you can just use the `rawValue` as the constant, rather than these properties. Having empty strings like this means you data modeling is insufficient in some way and can lead to subtle bugs.

Reply:
@jshier Question: Your last point is interesting to me. Won't I still need the empty strings for default cases when using raw values?

Reply:
Right, as long as the value you expect matches the case name or the manually assigned raw value.